pvlib/modelchain.py

```diff
@@ -295,7 +304,7 @@ def __init__(self, system, location,
                  solar_position_method='nrel_numpy',
                  airmass_model='kastenyoung1989',
                  dc_model=None, ac_model=None, aoi_model=None,
-                 spectral_model=None, temp_model='sapm',
+                 spectral_model=None, temp_model=None,
```

Comment:
should we change this to `temperature_model`? Would not be too bad to add a deprecation for `temp_model` because we're already allowing `**kwargs` here.

Reply:
I agree with making the change. How do I deprecate a change in a `ModelChain` attribute's name?

Reply:
In this case I think we have to manually test for the presence of `temp_model` in `**kwargs`.

```python
if temp_model in kwargs:
    warnings.warn('The temp_model keyword argument is deprecated. Use temperature_model instead', pvlibDeprecationWarning)
    if temperature_model is None:
        temperature_model = temp_model
    else:
        raise ValueError('only one of temperature_model and temp_model may be specified')
```